/smooth_local_planner:
  ros__parameters:
    debug: false

    targetTopic: "/goal_pose"          # 目标点话题名
    circleObs:  [  0.0,   0.0,   4.0, 
                  1.0,   8.0,   3.0, 
                  -8.0,   1.0,   2.0,
                  -5.0,  16.0,   5.0,
                  13.0,   2.0,   6.0,
                  -15.0, -12.0,   5.0,
                  -18.0,   6.0,   8.0,
                  12.0, -15.0,   4.0,
                  20.0,  19.0,   4.5,
                  -3.0, -18.0,   6.5]

    penaltyWeight: 1000.0              # 障碍权重
    pieceLength: 0.5                   # 路径切片长度
    relCostTol: 1.0e-6
